docs/content/direct/pre-reqs.md

```diff
@@ -53,18 +53,18 @@ follow [these instructions](https://github.com/kubestellar/kubeflex/blob/main/do
 
 ## Additional Software For Running the Examples
 
-- **argocd** - for the examples that use it
+- [**argocd**](https://argo-cd.readthedocs.io/en/stable/getting_started/) - for the examples that use it
 
 ## Additional Software For Building KubeStellar from Source
 
-- **go** version 1.20 or higher - to build Kubestellar
-- **make** - to build Kubestellar and create the Kubestellar image
-- **ko** - to create the Kubestellar image
+- [**go**](https://go.dev/doc/install) version 1.20 or higher - to build Kubestellar
+- [**make**](https://www.gnu.org/software/make/) - to build Kubestellar and create the Kubestellar image
+- [**ko**](https://ko.build/install/) - to create the Kubestellar image
```

Comment:
I would qualify the word image with "container", because we also use OCI images for Helm charts.